.github/workflows/build.yml

```diff
@@ -58,6 +58,11 @@ jobs:
         with:
           usesh: true
           prepare: |
+            mkdir -p /usr/local/etc/pkg/repos
+            sed 's/quarterly/latest/' /etc/pkg/FreeBSD.conf \
+              > /usr/local/etc/pkg/repos/FreeBSD.conf
+            pkg update -f
+            pkg upgrade -y
```

Comment:
Not necessary as nothing is installed by default. FreeBSD base itself isn't tracked by pkg [yet](https://wiki.freebsd.org/PkgBase).